src/linters/ArcanistESLintLinter.php

```diff
@@ -45,7 +45,7 @@ public function getDefaultBinary() {
     }
 
     public function getInstallInstructions() {
-        return pht('Install ESLint using `%s`.', 'npm install -g eslint eslint-plugin-react');
+        return pht('Install ESLint using `%s`.', 'npm install eslint');
```

Comment:
can you change this from `npm install eslint` to just `Make sure your project has ESLint installed.`.

I don't think `npm install eslint` is the proper recommendation here since we want to lock versions on that as well.